src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -203,6 +216,10 @@ public void onBuildComplete(AbstractBuild<?, ?> build,
             }
         }
 
+        if((showMatrixStatus && build.getProject() instanceof MatrixProject)){
```

Comment:
Should this be !showMatrixStatus &&?  Also you don't need the second set of parentheses.